go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/miekg/dns v1.1.14
 	github.com/openshift/coredns-mdns v0.0.0-20190710141149-66c708637267
 	github.com/opentracing/opentracing-go v1.1.0
+	github.com/openzipkin-contrib/zipkin-go-opentracing v0.3.5
 	github.com/openzipkin/zipkin-go-opentracing v0.3.5
```

Comment:
@cybertron is `github.com/openzipkin/zipkin-go-opentracing` still required?

Reply:
I can get rid of it if I change the import to reference the new location. I'll add that change to the PR.